main_2024.py

```diff
@@ -66,6 +68,8 @@ def main() -> int:
         print("Config key(s) not found")
         return -1
 
+    pathlib.Path(f"./{LOG_DIR}").mkdir(exist_ok=True)
```

Comment:
Use commas to separate directories: `pathlib.Path("parent_directory", "directory")`